lib/pure/hashes.nim

```diff
@@ -484,8 +503,9 @@ proc hash*[A](x: openArray[A]): Hash =
 proc hash*[A](aBuf: openArray[A], sPos, ePos: int): Hash =
   ## Efficient hashing of portions of arrays and sequences, from starting
   ## position `sPos` to ending position `ePos` (included).
+  ## The elements must have a hash proc.
   ##
-  ## ``hash(myBuf, 0, myBuf.high)`` is equivalent to ``hash(myBuf)``.
+  ## `hash(myBuf, 0, myBuf.high)` is equivalent to `hash(myBuf)`.
```

Comment:
Why these changes? They make no difference from what I've seen in the docgen.

Reply:
Ye, it doesn't make a difference in docgen, but single backticks are shorter and i think it looks cleaner (in the source). Also, the contributing guideline says:

> In nim sources, prefer single backticks to double backticks since it's simpler and `nim doc` supports it